conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -11,3 +11,4 @@
 
 # Add all the application routes to the prod.routes file
 ->         /                          prod.Routes
+GET        /test-only/                uk.gov.hmrc.trackingconsentfrontend.controllers.TestController.test
```

Comment:
Will this still work in QA?

Reply:
Its configurable with the `application.router` config property.
Do we need it to work in QA?

Reply:
Well yes ideally, for demonstration purposes?